writehat/static/js/engagementEdit.js

```diff
@@ -34,6 +34,32 @@ $(document).ready(function() {
 
   });
 
+  // Table Filtering
+  var status = $('input[name="status"]:checked');
+
+  // Custom range filtering function
+  $.fn.dataTable.ext.search.push(function (settings, data, dataIndex) {
+    var statusValue = new Array();
+    $.each($("input[name='status']:checked"), function() {
+      statusValue.push($(this).val());
+    });
+
+    var itemActive = data[4]; // use data for the active status column
```

Comment:
It may not be feasible now, but we should look for a way to add column definitions in the `DataTables` instantiation so we can refer to this by name instead of index. As it is today, adding a preceding column will break this function until the offset is updated.

Reply:
I looked a little bit into this but it seems all the documentation uses index as well. The alternative would probably be making an enum.

Reply:
Since datatables does not provide a good way to reference columns by name, this will have to do